packages/react-form/README.md

```diff
@@ -701,6 +701,16 @@ You can choose to initialize the list with an existing number of cards or no car
       <div>
         <Button onClick={() => removeItem(index)}>Remove</Button>
       </div>
+      <div>
+        <Button onClick={() => moveItem(index, index - 1)}>
```

Comment:
Since react-form will now blow up with an exception, it's the responsibility of the app to do the range checking. Untested suggestion:

```suggestion
        <Button disabled={index === 0} onClick={() => moveItem(index, index - 1)}>
```